packages/api/src/middleware/auth.ts

```diff
@@ -118,6 +118,12 @@ function authenticator(): RequestHandler {
       throw new ForbiddenError(`user is suspended`);
     }
 
+    if (user.disabled) {
+      throw new ForbiddenError(
+        `user is disabled - you maxed out your usage for the month, please input a payment method`
+      );
+    }
```

Comment:
Disabled users cannot be handled the same way as suspended users. They need specific handling on the operations we want to block. This will make the account unusable, just like for suspended, in which case there's no point in creating the new property.

Reply:
You can make checks about the user being disabled here instead:
- for live: https://github.com/livepeer/livepeer-com/blob/8e19a6f062630f8013f1fc769c0c9e04dd23b08b/packages/api/src/controllers/stream.ts#L1649
- for vod: https://github.com/livepeer/livepeer-com/blob/9411c8279173c7b68c31c92a7700399ec7ba40ac/packages/api/src/task/scheduler.ts#L346

Reply:
Moved this check there!